helm/templates/infrahub-server.yaml

```diff
@@ -60,7 +60,7 @@ spec:
               command:
                 - sh
                 - -c
-                - curl -s -f -o /dev/null http://localhost:{{ (index .Values.infrahubServer.ports 0).port }}/api/schema/summary || exit 1
+                - curl -s -f -o /dev/null http://localhost:{{ (index .Values.infrahubServer.ports 0).port }}/api/config || exit 1
```

Comment:
@fatih-acar do we need to update the version of the helm chart as well ?

Reply:
So you mean that it will be bumped regardless by some other process in Infrahub 1.1.5 and I don't need to touch anything now?

Reply:
Exactly